docs/sql-ref-syntax-qry-star.md

```diff
@@ -59,31 +59,31 @@ except_clause
 
 ```sql
 -- Return all columns in the FROM clause
-SELECT * FROM VALUES(1, 2) AS TA(c1, c2), VALUES(‘a’, b’) AS TB(ca, cb);
```

Comment:
interesting, did the test pass before?

Reply:
Hmm..., I think `some docs` under `docs\...` are not tested.

Reply:
oh sorry I misread it, it's doc only not a test